app/assets/stylesheets/reference.scss

```diff
@@ -339,3 +339,7 @@ p.quickref {
   margin-bottom: 20px;
   background-color: $callout-color;
 }
+
+.all-commands {
+  text-align: right;
```

Comment:
not sure if this is enough. I'm seeing this: 
![image](https://user-images.githubusercontent.com/1999050/68708768-4df9d380-058c-11ea-8a66-15b1de46db81.png)

Reply:
and just like you did in the other PR, you need to precompiling the CSS (this css was not applied on the review app https://git-scm-pr-1397.herokuapp.com/docs )

Reply:
You are right, I forgot to precompile for this one. 

When you say not enough, what do you mean exactly? I was not sure of the best way to show the “All commands” link (I’m really not a designer :p). 
If you or others have other ideas that would be more pleasing visually I’ll gladly update this branch.

Reply:
not really a designer either 😅 not sure, but I was expecting the 'All commands' to be aligned under the right column ? seems weird to be that isolated (once again, my word is not very valuable in design questions...)

Reply:
I agree the design is a work in progress. I did not want it just below the "Plumbing commands" list because I don't want users to think that this link will get them to the full list of plumbing commands, rather then then full list of all commands.

Reply:
I actually think it looks better without the right alignment, i.e. as it is on heroku right now :
https://git-scm-pr-1397.herokuapp.com/docs